tests/test_api_longtests.py

```diff
@@ -51,3 +51,21 @@ def test_list_issue(instance):
                            "We will be too many to be listed on one page")
     issues = repo.get_issues()
     assert len(issues) > 98
+
+
+def test_list_team_members(instance):
+    org = Organization.request(instance, test_org)
+    team = org.create_team(test_team, "Team for longtests")
+    users = []
+    for i in range(100):
+        users.append(
+            instance.create_user(
+                test_user + str(i),
+                test_user + str(i) + "@example.org",
+                "abcdefg1.23AB",
+                send_notify=False
+            ),
+        )
+    for user in users:
+        team.add_user(user)
+    assert len(team.get_members()) == len(users)
```

Comment:
Do we test deleting users and getting the length?
I know the AllSpice instance gets cleaned up at the end, but might be nice to test removing users and then deleting the team and ensuring the team is gone.

Reply:
The right place for that would be the "unit" tests, which are in [this file](https://github.com/AllSpiceIO/py-allspice/blob/main/tests/test_api.py). We have tests for deleting the team, adding a member etc, but there's none for removing a user. I can add that in along with this repo.